contracts/kleros/KlerosGovernor.sol

```diff
@@ -193,42 +215,51 @@ contract KlerosGovernor is Arbitrable {
      *  @param _dataSize List of lengths in bytes required to split calldata for its respective targets.
      *  @param _description String in CSV format that describes list's transactions.
      */
-    function submitList(address[] _target, uint[] _value, bytes _data, uint[] _dataSize, string _description) public payable duringSubmissionPeriod {
+    function submitList (address[] _target, uint[] _value, bytes _data, uint[] _dataSize, string _description) public payable duringSubmissionPeriod {
         require(_target.length == _value.length, "Incorrect input. Target and value arrays must be of the same length.");
         require(_target.length == _dataSize.length, "Incorrect input. Target and datasize arrays must be of the same length.");
-        require(msg.value == submissionDeposit, "Submission deposit must be paid in exact amount.");
+        require(msg.value >= getSubmissionCost(), "Submission deposit must be paid in full.");
         Session storage session = sessions[sessions.length - 1];
         Submission storage submission = submissions[submissions.length++];
         submission.submitter = msg.sender;
-        submission.deposit = submissionDeposit;
-        bytes32 listHash;
-        bytes32 prevTxHash;
-        bytes32 currentTxHash;
+        submission.deposit = getSubmissionCost();
```

Comment:
Second call to getSubmission, we can save the value which is cheaper in gas and simple (easy guarantee that the value won't change).

Reply:
There is a "stack too deep" error. So we can't add any more variables to the function